src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../Types/user';
+
+interface Props {
+  user?: User | null;
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
+    </a>
```

Comment:
[REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has `null` or `undefined` value. In this case, you're using optional chaining to handle the possibility of `user` being `null` or `undefined`. Instead, you should conditionally render the `UserInfo` component in the parent component, only if `user` has a value. This way, you avoid unnecessary checks and potential render of an empty anchor tag.